WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingHomeViewModel.kt

```diff
@@ -50,9 +50,12 @@ class PrepublishingHomeViewModel @Inject constructor(
                 HomeUiState(
                         actionType = PUBLISH,
                         actionResult = editPostRepository.getPost()?.let { postImmutableModel ->
-                            UiStringText(
-                                    postSettingsUtils.getPublishDateLabel(postImmutableModel)
-                            )
+                            val label = postSettingsUtils.getPublishDateLabel(postImmutableModel)
```

Comment:
I added this here because it's related to the visibility. When you press publish and the contents of the editor is empty and you change the visibility to `Private` the `Publish Date` is blank so this is to ensure that the default value of `Immediately` is set.